frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -7,80 +9,164 @@ import {
+            allowConnect
+              ? StripeConnectButtonStates.ENABLED
+              : StripeConnectButtonStates.DISABLED
+          }
+        />
+      </>
+    )
+  } else {
+    return (
+      <>
+        <InlineMessage variant="info" my="2rem">
+          <Text>
+            You are currently in test mode. You can choose to skip connecting a
+            Stripe account after clicking the button below.
+          </Text>
+        </InlineMessage>
+        <StripeConnectButton connectState={StripeConnectButtonStates.ENABLED} />
+      </>
     )
   }
+}
 
-  if (hasPaymentCapabilities) {
-    return (
-      <Skeleton isLoaded={!isLoading}>
-        <Flex mb="2.5rem">
-          <Icon
-            aria-hidden
-            marginEnd="0.5em"
-            color="success.500"
-            fontSize="1rem"
-            h="1.5rem"
-            as={BxsCheckCircle}
-            mr={2}
-          />
-          <Text>Your Stripe account is linked to FormSG</Text>
-        </Flex>
-      </Skeleton>
+const ConnectionStatusText = ({
+  color,
+  icon,
+  text,
+}: {
+  color: string
+  icon: As
+  text: string
+}) => (
+  <>
+    <Icon
+      aria-hidden
+      marginEnd="0.5em"
+      color={color}
+      fontSize="1rem"
+      h="1.5rem"
+      as={icon}
+      mr={2}
+    />
+    <Text>{text}</Text>
+  </>
+)
+
+const AfterConnectionInfo = ({
+  isProductionEnv,
+}: {
+  isProductionEnv: boolean
+}): JSX.Element => {
+  const { hasPaymentCapabilities, isLoading, isError } = useAdminFormPayments()
+
+  let connectionInfo: JSX.Element
+
+  if (isError) {
+    // Base case: Error retrieving form payments data
+    connectionInfo = (
+      <ConnectionStatusText
+        color="danger.500"
+        icon={BxsError}
+        text="Something went wrong when validating the connected Stripe account."
+      />
     )
+  } else if (isProductionEnv) {
+    if (hasPaymentCapabilities) {
+      // Live mode: Account connected successfully and can be charged
+      connectionInfo = (
+        <ConnectionStatusText
+          color="success.700"
+          icon={BxsCheckCircle}
+          text="Your Stripe account is connected to this Form."
+        />
+      )
+    } else {
+      // Live mode: Linked account has no payment capabilities.
+      connectionInfo = (
+        <ConnectionStatusText
+          color="warning.500"
+          icon={BxsInfoCircle}
+          text="The connected account does not have the ability to process payments."
+        />
+      )
+    }
+  } else {
+    if (hasPaymentCapabilities) {
+      // Test mode: Account connected successfully but note that will only be on test mode
+      connectionInfo = (
+        <ConnectionStatusText
+          color="success.700"
+          icon={BxsCheckCircle}
+          text="Stripe account connected. Payments made on this form will only show in test mode in your Stripe account."
+        />
+      )
+    } else {
+      // Test mode: Stripe account connection step skipped
```

Comment:
There could be a case where their connected account doesn't have payment capabilities right? Is this still the message we should show them? (i.e. not prod -> connect account -> connected account doesn't have payment cap -> "You are connected to a test account")

Reply:
Yep! This is the case when they skip the account connection in test mode. The design is here: https://www.figma.com/file/CMq0CJtawn7NGM7Zdpi8Ti/Form-Design-Master?type=design&node-id=17226-180896&t=zFJcUOjCPFQtGK9F-0